controllers/keyword.go

```diff
@@ -22,6 +24,13 @@ const (
 	uploadSuccessFlash = "CSV uploaded successfully"
 )
 
+var FilterList = []map[string]string{
+	{
+		"queryString": "filter[keyword]",
+		"filter":      "keyword",
```

Comment:
At first I thought that there were two filters but I was wrong since there is only only `filter[keyword`. The second item is used to generate the model condition. So It's a bit hard to understand that from the get-go.

How about renaming the second key to `modelCondition` so that it's more explicit:

```suggestion
		"modelCondition":      "keyword",
```

Reply:
That is a good idea. Fixed in d2064d29cb90fca0dfc74b75bf9ced442e04278a